src/Psl/Xml/Issue/Level.php

```diff
@@ -0,0 +1,126 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Psl\Xml\Issue;
+
+use Psl\Exception\InvariantViolationException;
+
+use function in_array;
+use function Psl\invariant;
+
+use const LIBXML_ERR_ERROR;
+use const LIBXML_ERR_FATAL;
+use const LIBXML_ERR_NONE;
+use const LIBXML_ERR_WARNING;
+
+/**
+ * @psalm-immutable
+ */
+final class Level
```

Comment:
Made more sense to convert the level logic into it's own VO.

Reply:
agree!